src/frontends/tensorflow/include/openvino/frontend/tensorflow/graph_iterator.hpp

```diff
@@ -5,13 +5,15 @@
 #pragma once
 
 #include "openvino/core/any.hpp"
-#include "tensorflow_frontend/decoder.hpp"
-#include "tensorflow_frontend/utility.hpp"
+#include "openvino/frontend/tensorflow/decoder.hpp"
+#include "openvino/frontend/visibility.hpp"
 
 namespace ov {
 namespace frontend {
+namespace tensorflow {
+
 /// Abstract representation for an input model graph that gives nodes in topologically sorted order
-class TF_API GraphIterator : ov::RuntimeAttribute {
+class TENSORFLOW_API GraphIterator : ov::RuntimeAttribute {
```

Comment:
will not this macro conflict with actual TF macro? 
Would be better to have name like OV_TENSORFLOW_API  to denote it's OV specific

Reply:
But maybe we just need to remove a lot of macro per library and use generic `OPENVINO_API`

Reply:
>> will not this macro conflict with actual TF macro? 
what do you mean? I renamed TF macro to TENSORFLOW macro